stdlib/@python2/rlcompleter.pyi

```diff
@@ -1,7 +1,9 @@
-from typing import Any, Union
+from typing import Any, TypeVar, Union
```

Comment:
```suggestion
from typing import Any, Union
```

Doesn't look like this is used anywhere (not picked up by CI because only a subset of our CI runs on the Python 2 stubs)